### PR TITLE
Added Stale application

### PR DESCRIPTION
**Pull request by Nyholm**
This will close issues that does not have any activity in 4 months.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Maybe go for 12 months?

**Comment by Nyholm** [comment]
Default was 2 months =)

If there is no activity at all (even tough there is a reminder) for 4 months, then the issue is no important enough... One could just add a comment after the reminder and it will be open for another 4 months.

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+daysUntilStale: 120
+daysUntilClose: 14
+#exemptLabels:
+# Label to use when marking an issue as stale
+staleLabel: lifecycle/stale
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
```
Comment (sagikazarmark) [inline .github/stale.yml:9]: Maybe add when it will be closed?

**Review by sagikazarmark — APPROVED** [review]

**Comment by Nyholm** [comment]
Thank you

**Pull request merged**